test/tests/functional/pbs_cray_pagg_id.py

```diff
@@ -42,14 +42,23 @@
 @tags('cray')
 class TestCrayPaggIdUniqueness(TestFunctional):
     """
+    This test suite is written to verify that the PAGG ID provided to ALPS
+    while confirming and releasing an ALPS reservation is not equal to the
+    session ID of the job.
+    This test is specific to Cray and will also not work on the Cray as well,
```

Comment:
You want to say it will not work on Cray simulator, right?

Reply:
yes, I will change it...